constants.py

```diff
@@ -112,3 +112,4 @@ class ServoExtension(Enum):
 # The altitude at which the rocket is expected to reach apogee, without the airbrakes
 TARGET_ALTITUDE = 1700  # m (5,100 ft)
 CURVE_FIT_INITIAL = [-10.5, 0.03]
+APOGEE_PREDICTION_FREQUENCY = 100  # packets
```

Comment:
this can and should me MUCH lower imo

Reply:
when we looked at CPU  usage, it was like 3% with this. If we lower this to 2 packets, CPU usage still is only ~20%. This change could get us a prediction like .1 or .2 seconds faster, and we really need every tenth of a second we can get. Ideally we can lower the prediction time to ~1.5 seconds.

Reply:
well I'd be down to lower it to 50 packets, but anything lower is just useless compute imo

Reply:
but like why not do it faster? Thats still a .1 second delay. Like I would get it if we needed the extra performance, but we don't--we're not even close to hitting it. 

The faster we computer it, the more frequently/faster we can check it converges.

Reply:
I'd lower it to around 10 data packets maybe. Even though the speed of convergence is pretty much negligible, it would be good to log it, and more data points gives better post-processing

Reply:
alrightttt i'll cave